packages/lodestar/src/chain/forkChoice/arrayDag/lmdGhost.ts

```diff
@@ -379,6 +379,18 @@ export class ArrayDagLMDGHOST extends (EventEmitter as { new(): ForkChoiceEventE
     return (node)? this.toBlockSummary(node) : null;
   }
 
+  public getBlockSummaryByParentBlockRoot(blockRoot: Uint8Array): BlockSummary[] {
```

Comment:
We indexed everything in forkchoice already so should be able to get parent by `nodeIndices` and `nodes` then we can get all child of that found node.